central/graphql/resolvers/image_vulnerabilities.go

```diff
@@ -1176,31 +1176,33 @@ func (resolver *imageCVEV2Resolver) ExceptionCount(ctx context.Context, args str
 	return int32(count), nil
 }
 
-func (resolver *imageCVEV2Resolver) Advisory(ctx context.Context) (string, error) {
+func (resolver *imageCVEV2Resolver) Advisory(ctx context.Context) (*advisoryResolver, error) {
 	defer metrics.SetGraphQLOperationDurationTime(time.Now(), pkgMetrics.ImageCVEs, "Advisory")
 	if resolver.ctx == nil {
 		resolver.ctx = ctx
 	}
 
 	// Short path. Full image is embedded when image scan resolver is called.
 	if embeddedVuln := embeddedobjs.VulnFromContext(resolver.ctx); embeddedVuln != nil {
-		return embeddedVuln.GetAdvisory().GetName(), nil
+		advisory, _ := resolver.root.wrapAdvisory(embeddedVuln.GetAdvisory(), true, nil)
```

Comment:
can you just do a return of the call to the resolver so we don't lose the error?